### ISSUE TITLE
preference to NOT display ADF dialog on drag and drop

### ISSUE THREAD

**Issue by DannyCork** [dirkwhoffmann/vAmiga#515]
I'm testing a lot of disks and everyone I reset vAmiga and drop an add image I get a dialog like this

![image](https://user-images.githubusercontent.com/3512672/125006212-813c8080-e055-11eb-9b40-ad019b030251.png)

The dialog is of little value to be honest. Is it possible to skip this? Perhaps a preference checkbox?

**Comment by dirkwhoffmann** [comment]
>  Is it possible to skip this? Perhaps a preference checkbox?

Sorry for replying late. I am busy working on a base Mandelbrot generator which I want to use for student projects in future. I need to finish this up before the winter term starts. 

To avoid the dialog, you can make a drive a "drop target".

<img width="164" alt="Bildschirmfoto 2021-07-13 um 17 17 31" src="https://user-images.githubusercontent.com/12561945/125478514-8cfbc47c-9c92-4ef9-8e3f-b7c73263d1e6.png">

Once a drop target is set, any ADF which is dragged in will be inserted automatically into the corresponding drive.

**Comment by dirkwhoffmann** [comment]
I reopened this issue because I think I have found a way to make drag & drop more efficient. There is already a prototype implementation for VirtualC64 and I plan to port it to vAmiga in the near future. In the next version of VirtualC64, when a media file is dragged in, four drop zones will pop up (for drive 8, drive 9, the cartridge port, and the datasette). 

<img width="858" alt="Bildschirmfoto 2021-08-05 um 15 36 35" src="https://user-images.githubusercontent.com/12561945/128362868-d2814015-d7a5-48be-8638-864174a0f83c.png">

In the image above, only the drop some for drive 8 is enabled, because the file is of type D64 and drive 9 is disconnected. 

If the file is dropped onto the zone for drive 8, it is directly inserted into drive 8. If it is dropped somewhere else (outside any drop zone), the media dialog appears. In vAmiga, there is neither a cartridge port nor a datasette, but we'll have four drop zones, too (for df0, df1, df2, and df3). 

I tried to implement similar functionality in the past, but I always failed due to technical difficulties. This time, I think I nailed it. 

The "drop target" feature will be deleted as it will no longer be needed.
